cgmanifest.json

```
@@ -15584,8 +15584,8 @@
         "type": "other",
         "other": {
           "name": "OpenIPMI",
-          "version": "2.0.25",
-          "downloadUrl": "https://sourceforge.net/projects/openipmi/files/latest/download/openipmi-2.0.25.tar.gz"
+          "version": "2.0.32",
+          "downloadUrl": "http://downloads.sourceforge.net/openipmi/OpenIPMI-2.0.32.tar.gz"
```

NIT: lets convert `s/http/https/g`
```suggestion
          "downloadUrl": "https://downloads.sourceforge.net/openipmi/OpenIPMI-2.0.32.tar.gz"
```